src/components/ImageDetails/ImageDetails.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input, Box, CloseButton } from '@chakra-ui/react';
+
+function ImageDetails({ index, name, preview, setFilesIntermediate }) {
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'index' is defined but never used.